semgrep_runner_v1.proto

```diff
@@ -0,0 +1,38 @@
+syntax = "proto3";
+
+package semgrep_runner_v1;
+
+option go_package = "semgrep_runner.v1";
+
+import "protos/semgrep_output_v1.proto";
+
+enum RuleFormat {
+  YAML = 0;
+  JSON = 1;
+  JSONNET = 2;
+}
+
+message FileBlob {
+    string file_path = 1;
```

Comment:
Is there a way to have stronger typing for file path? Can you define an alias like
fpath which resolve to a string, but it's still different than a string.

Reply:
Yes! Added a `FilePath` message type (which simply contains a string called `path`)